sim/common/cata/other_effects.go

```diff
@@ -8,6 +8,14 @@ import (
 	"github.com/wowsims/cata/sim/core/stats"
 )
 
+type ItemVersion int32
+
+const (
+	ItemVersionLFR    ItemVersion = 0
+	ItemVersionNormal ItemVersion = 1 << iota
+	ItemVersionHeroic
+)
```

Comment:
Minor optimization, but you can do this instead:
```
type ItemVersion byte

const (
	ItemVersionLFR ItemVersion = iota
	ItemVersionNormal
	ItemVersionHeroic
)
```

This also lets you directly index arrays by it below.